### PR TITLE
Added new logo placeholder for web interface header logo.

### PR DESCRIPTION
**Pull request by emanuelschuetze**
@FinnStutzenstein Can you please add a commit which adds a logo placeholder also for motion / assignment pdf ballot paper. I tried it but I have not yet completely understand how and where to use the ImageConverter promises...

### CODE REVIEW

File: openslides/core/config_variables.py [hunk 1]
```diff
@@ -270,3 +271,13 @@ def get_config_variables():
...
         group='Logo',
         hidden=True)
+
+    yield ConfigVariable(
+        name='logo_web_header',
+        default_value={
+            'display_name': 'Web interface header logo',
+            'path': ''},
+        input_type='logo',
+        weight=311,
```
Comment (normanjaeckel) [inline openslides/core/config_variables.py]: Please use new weight number. (I know that this is hidden but this may change in future. :-)

Comment (FinnStutzenstein) [inline openslides/core/config_variables.py]: Done.

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze Logos now work; But you need to change the header function to your needs. I just copied it from the intevation/dgb branch and fit our logo system into.

File: openslides/assignments/static/js/assignments/pdf.js [hunk 2]
```diff
@@ -266,14 +266,46 @@ angular.module('OpenSlidesApp.assignments.pdf', ['OpenSlidesApp.core.pdf'])
...
-    function($filter, gettextCatalog, PDFLayout, Config, User) {
+    'ImageConverter',
+    function($q, $filter, gettextCatalog, PDFLayout, Config, User, ImageConverter) {
         var createInstance = function(assignment, poll, pollNumber) {
 
+            var logoAssignmentBallotPaperUrl = Config.get('logo_pdf_assignment_ballot_paper').value.path;
+            var imageMap = {};
+
+            // PDF header
+            var header = function() {
```
Comment (FinnStutzenstein) [inline openslides/assignments/static/js/assignments/pdf.js]: Please check the layout etc.

File: openslides/motions/static/js/motions/pdf.js [hunk 3]
```diff
@@ -547,25 +549,58 @@ angular.module('OpenSlidesApp.motions.pdf', ['OpenSlidesApp.core.pdf'])
         */
         var createInstance = function(title, id) {
 
+            var logoMotionPollUrl = Config.get('logo_pdf_motion_poll').value.path;
+            var imageMap = {};
+
+            // PDF header
+            var header = function() {
```
Comment (FinnStutzenstein) [inline openslides/motions/static/js/motions/pdf.js]: Here too.

**Comment by FinnStutzenstein** [comment]
...and two missing semicolons

**Comment by emanuelschuetze** [comment]
@FinnStutzenstein All done now. Can you please review. Thanks.

File: openslides/core/static/templates/index.html [hunk 4]
```diff
@@ -29,7 +29,8 @@
     <!-- Logo -->
     <div class="title">
       <a ui-sref="home">
-        <img src="/static/img/openslides-logo-dark.png" alt="Logo" height="35">
+        <img src="/static/img/openslides-logo-dark.png" alt="OpenSlides Logo" height="35">
```
Comment (emanuelschuetze) [inline openslides/core/static/templates/index.html]: Yes, I'll keep the software and brand name "OpenSlides"!

Comment (FinnStutzenstein) [inline openslides/core/static/templates/index.html]: Ok ;)

**Comment by emanuelschuetze** [comment]
Already approved. All done. Ready for merge...

**Pull request merged**